lib/postoffice/messaging/pending_message.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Postoffice.Messaging.PendingMessage do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "pending_messages" do
+    belongs_to :message, Postoffice.Messaging.Message
+    belongs_to :publisher, Postoffice.Messaging.Publisher
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(pending_message, attrs) do
```

Comment:
I don't add `unique key` validation because the clients can send the same message N times